x/incentives/keeper/gauge_test.go

```diff
@@ -46,6 +48,9 @@ func (suite *KeeperTestSuite) TestNonExistentDenomGaugeCreation() {
 // TODO: Make this test table driven
 // OR if it needs to be script based,
 // remove lots of boilerplate so this can actually be followed
+// TODO: Make issue for this test
+
+// Tests non perpetual gauge distribution logic.
 func (suite *KeeperTestSuite) TestNonPerpetualGaugeOperations() {
```

Comment:
TODO: Make issue for this test

Reply:
This test is being completely refactored to be table-driven in this PR: https://github.com/osmosis-labs/osmosis/pull/1932